openmetadata-service/src/main/java/org/openmetadata/service/migration/MigrationUtil.java

```diff
@@ -164,13 +162,19 @@ public static <T extends EntityInterface> void readAndProcessEntity(
               withName
                   ? FullyQualifiedName.buildHash(EntityInterfaceUtil.quoteName(entity.getFullyQualifiedName()))
                   : FullyQualifiedName.buildHash(entity.getFullyQualifiedName());
-          upsertBatch.bind("nameHashColumnValue", hash).bind("id", entity.getId().toString()).add();
+          int result =
+              handle
+                  .createUpdate(updateSql)
+                  .bind("nameHashColumnValue", hash)
+                  .bind("id", entity.getId().toString())
+                  .execute();
+          if (result <= 0) {
+            LOG.error("No Rows Affected for Updating Hash with Entity Name : {}", entity.getFullyQualifiedName());
```

Comment:
if result is zero should we retry? if there is no exception

Reply:
it will again come in further reads with ->  select * from <entity_table> where nameHash = '' or nameHash is null;